example/instruction_tuning_pipeline/finetune_clm.py

```diff
@@ -700,12 +704,7 @@ def concatenate_data(dataset, max_seq_length):
 
     if finetune_args.delta:
         if finetune_args.resume_peft != "":
-            if finetune_args.delta == "ssf":
-                peft_config = SSFConfig(target_modules=deltatuner_args.ssf_target_modules,
-                bias="none",
-                task_type="CAUSAL_LM",
-                )
-            model = DeltaTunerModel.from_pretrained(model, finetune_args.resume_peft, config=peft_config, denas_config=deltatuner_args)
+            model = DeltaTunerModel.from_pretrained(model, finetune_args.resume_peft, denas_config=deltatuner_args)
```

Comment:
@zhouyu5 , can you add an example to show denas_config, since you're merging peft and deltatuner here. What is now the 'deltatuner_args' looks like?